### DISCUSSION TITLE
DKIM Reporting -- domain?

### DISCUSSION THREAD

**Discussion by christianreiss** [rspamd/rspamd#4110] [Q&A]
Hey folks,

I am a little bit puzzled about the required dmarc.reporting.domain variable/parameter, domain in dmarc.
I am running a multitude of domains on my email server and for all of those I would like to send reports as.

I also would like to serve all dmark/spf domains out there which had an email delivery to my host.
So I the only case I would like to do is:

"someone sending an authorized or not authorized email as $source_domain to any of my $local_domains. In accordance with dmarc setting in DNS the rua/ruf of $source_domain notified, the sender is dmarc->email variable."

Where does dmarc.reporting.domain fit in there? Does it limit either side? To what cause?
How can I set the reporting module to the case above?

What am I getting wrong here?
-Chris.

**Comment by alainwolf** [comment]
You can see the created reports if called with the `--verbose` option:

    rspamadm dmarc_report -v

By looking at the XML you will see that reports for all your hosted domains will be created, The `domain`  and  `org_name` configuration parameters just defines which organization is sending out reports on behalf of the hosted domains.
